Library/Formula/ansible.rb

```diff
@@ -204,6 +204,14 @@ class Ansible < Formula
     sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
   end
 
+	#
+	# apache libcloud (for Google GCE cupport)
+	#
```

Comment:
This is fixed.  Do I need to submit a new pull request?


Robert Marsa | Senior Staff Software Engineer | rmarsa@google.com <mailto:rmarsa@google.com> | 

> On Feb 2, 2016, at 3:48 AM, Mike McQuaid <notifications@github.com> wrote:
> 
> In Library/Formula/ansible.rb <https://github.com/Homebrew/homebrew/pull/48712#discussion_r51545263>:
> 
> > @@ -204,6 +204,14 @@ class Ansible < Formula
> >      sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
> >    end
> >  
> > +	#
> > +	# apache libcloud (for Google GCE cupport)
> > +	#
> Indentation here needs fixed.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/pull/48712/files#r51545263>.
>